frontend/src/pages/CREResources/AddImagePage/AddNewImageForm.tsx

```diff
@@ -169,6 +190,28 @@ export const AddNewImageForm: React.FC = () => {
           valid = false;
         }
         break;
+        case 'git':
+          if (formState.state.name.length > 0) {
+            formDispatch({ type: 'validate', key: 'name', value: true });
+          } else {
+            formDispatch({ type: 'validate', key: 'name', value: false });
+            valid = false;
+          }
+  
+          if (formState.state.repository.length > 0) {
+            formDispatch({ type: 'validate', key: 'repository', value: true });
+          } else {
+            formDispatch({ type: 'validate', key: 'repository', value: false });
+            valid = false;
+          }
+
+          if (formState.state.repository.length > 0) {
```

Comment:
I believe this was meant to be:

```suggestion
          if (formState.state.gitRef.length > 0) {
```

However, as mentioned above, this is an optional field, so this validation should not be there.

I don't think we can easily validate the value provided here (i.e. ensure it's a valid git ref within the repo - this would require looking at the repo), so probably just remove the validation alltogether